pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -549,6 +551,15 @@ public Object getObject(String type, String value, byte[] byteValue) throws SQLE
       }
     }
 
+    if (type.equals("inet")) {
+      try {
+        return InetAddress.getByName(value);
+      } catch (UnknownHostException e) {
+        throw new PSQLException(GT.tr("Return Inet address is not supported.", type),
```

Comment:
This is a confusing message. If it is indeed an unknown host, why return Inet address not supported ?

Reply:
Oh! I am sorry for my mistake. It should be `IP address of a host could not be determined`